setup.py

```diff
@@ -19,6 +19,7 @@ def read_file(filename):
     'waitress',
     'cliquet>=2.13,<3',
     'jsonschema',
+    'functools32',  # for old pip versions.
```

Comment:
I would rephrase the comment, because if old version of pip doesn't includes functools32 and we need it, then it is a kinto dependency right?